test/book_finder_test.dart

```diff
@@ -111,4 +111,55 @@ void main() {
     expect(book.info.industryIdentifiers[0].type, 'ISBN_10');
     expect(book.info.industryIdentifiers[0].identifier, '1933820217');
   });
+
+  test('Test QueryTypes', () async {
+    ///  .intitle
+    List<Book> books = await queryBooks(
+      'harry potter and the philosopher\'s stone',
+      queryType: QueryType.intitle,
+      maxResults: 1,
+      printType: PrintType.books,
+      orderBy: OrderBy.relevance,
+    );
+
+    expect(books[0].info.title.toLowerCase(),
+        "harry potter and the philosopher's stone");
+
+    /// .inauthor
+    books = await queryBooks(
+      'J. K. Rowling',
+      queryType: QueryType.inauthor,
+      maxResults: 1,
+      printType: PrintType.books,
+      orderBy: OrderBy.relevance,
+    );
+
+    expect(books[0].info.authors[0].toLowerCase(), "j. k. rowling");
```

Comment:
```suggestion
    expect(books.first.info.authors[0].toLowerCase(), "j. k. rowling");
```

Use `.first` instead of `[0]` to keep the code clean. Same with the ones below